### ISSUE TITLE
Multiselect with shift key

### ISSUE THREAD

**Issue by brechtvhb** [harvesthq/chosen#135]
The multi select should allow us to select more then items at once by holding the shift key, like the regular non enhanced version does.

**Comment by casph** [comment]
+1

**Comment by patawa91** [comment]
I'm working on a pull request right now that allows user shift click and highlight multiple. Then the second click adds the highlighted items. Has a pull request for this type of thing been submitted? Is there even a possibility of this functionality being pulled in?

**Comment by kevin-brown** [comment]
> Has a pull request for this type of thing been submitted?

Yup, there's one at https://github.com/harvesthq/chosen/pull/1563

**Comment by laantorchaweb** [comment]
be aware, my PR for adding multiselection holding shift key is from 2013, I had rebased the code and updated it last week so it's ok to merge but I don't think that this will happen any time soon. This issue is from 2011...

**Comment by NeumannPSG** [comment]
+1

**Comment by NeumannPSG** [comment]
+1 for this feature

**Comment by cahrens** [comment]
+1 also for this
